src/components/Sidebar/ScenePane.jsx

```diff
@@ -37,7 +40,7 @@ function ScenePane({ closeCallback }) {
   }, shallowEqual);
```

Comment:
This shallowEqual will only compare the reference of the array afaik. Which will always be unique since the filter creates a new array. Maybe you meant to use shallowEqualArrays?